server_manager/www/ui_components/outline-server-view.ts

```diff
@@ -844,6 +850,7 @@ export class ServerView extends DirMixin(PolymerElement) {
   selectedPage: 'progressView' | 'unreachableView' | 'managementView' =
     'managementView';
   selectedTab: 'connections' | 'settings' = 'connections';
+  featureFlags = {};
```

Comment:
Considering initializing with the flag that this component uses:

```suggestion
  featureFlags = {serverMetricsTab: false};
```